projects/bazel-rules-fuzzing-test/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+
+RUN apt-get update && apt-get -y install \
+    python wget
```

Comment:
python3 is already there in image, no need to install.
wget should be there. if missing, add it here - https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile

Reply:
> python3 is already there in image, no need to install.

This is actually a workaround for a Bazel quirk, which requires `/usr/bin/python` to be present on the system. The `python3` package alone does not provide that and results in the `bazel` tool not working inside `build.sh`. This is documented in bazelbuild/bazel#8665 and bazelbuild/bazel#8685.

We can also try to create a symlink from `/usr/bin/python` to `/usr/bin/python3` - is there a way we can do it directly using `apt-get`?

Let me know if you'd prefer we go down this path. We can also install both `python` and `python3` in the base image.

> wget should be there. if missing, add it here - https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/Dockerfile

It is missing - let me add it there in a separate PR.

Reply:
For the record, this was addressed in #4976.